### PR TITLE
Avatar component added

### PR DESCRIPTION
**Pull request by kr-sarvesh**
Please review

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+@import url('/css/style.css');
+
+body {
+  background-image: none;
+}
+.wrapper .bottom-nav {
+  width: 100%;
+  background: #f9f9f9;
```
Comment (vikrantkode) [inline components/alert/alert.css]: declare variables and use here. Don't hard-code .

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  line-height: 1.5;
+}
+
+/* alert styles starts here */
+
+.alert {
+  font-size: 1.3rem;
+  width: 80%;
+  padding: 1rem;
+  color: #f9f9f9;
```
Comment (vikrantkode) [inline components/alert/alert.css]: same here

File: components/alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>alert</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="/prism.css" />
+    <link rel="stylesheet" href="all.min.css" />
+    <link rel="stylesheet" href="./alert.css" />
```
Comment (vikrantkode) [inline components/alert/alert.html:11]: put all the css files in one and import from there

**Review by vikrantkode — CHANGES_REQUESTED** [review]
- Changes Suggested . Have a look on them
- Can see some empty `BIN` files. 
- Please add screenshots of the pages u made if not deployed.
- Rest is good.

File: components/alert/alert.css (hunk 1 above)
Comment (kr-sarvesh) [reply components/alert/alert.css]: Thank you Vikrant for your vaulable feedback

**Pull request closed without merging**